arm64环境编译因为zlib无法下载出现异常

问题描述：
根据github首页的arm64的编译方法，在ubuntu16.04，arm64环境进行编译
```bash
root@arm13:~/cubefs/cubefs-master# export CPUTYPE=arm64_gcc4 && bash ./build.sh
```
出现中断
```bash
Resolving codeload.github.com (codeload.github.com)... 20.205.243.165
Connecting to codeload.github.com (codeload.github.com)|20.205.243.165|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 320958 (313K) [application/x-gzip]
Saving to: 'v1.9.3'

v1.9.3                               100%[===================================================================>] 313.44K  1.30MB/s    in 0.2s

2022-07-14 09:39:32 (1.30 MB/s) - 'v1.9.3' saved [320958/320958]

cp: cannot stat '/root/cubefs/cubefs-master/vendor/dep/zlib-1.2.11/*': No such file or directory
build Zlib...
build/build.sh: line 59: ./configure: No such file or directory
make[1]: Entering directory '/root/cubefs/cubefs-master/build/out/zlib'
make[1]: *** No targets specified and no makefile found.  Stop.
```
查看日志
```bash
build linux arm64 gcc4.9
--2022-07-14 09:38:56--  http://www.zlib.net/zlib-1.2.11.tar.gz
Resolving www.zlib.net (www.zlib.net)... 85.187.148.2
Connecting to www.zlib.net (www.zlib.net)|85.187.148.2|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-07-14 09:38:57 ERROR 404: Not Found.

tar (child): zlib-1.2.11.tar.gz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```
应该是下载zlib的地址过期了，所以没下载下来，出现异常

补充原因：

https://zlib.net/zlib-1.2.11.tar.gz 地址已经修改为
https://zlib.net/fossils/zlib-1.2.11.tar.gz

相关issue：https://github.com/madler/zlib/issues/649

所以相关的脚本引用的地址需要更新

--- a/build.sh
+++ b/build.sh
@@ -54,7 +54,7 @@ get_rocksdb_compress_dep() {
     cd ${RootPath}/vendor/dep
 
     if [ ! -d "${RootPath}/vendor/dep/zlib-1.2.11" ]; then
-        wget http://www.zlib.net/zlib-1.2.11.tar.gz
+        wget https://zlib.net/fossils/zlib-1.2.11.tar.gz
         tar zxf zlib-1.2.11.tar.gz
     fi
 

